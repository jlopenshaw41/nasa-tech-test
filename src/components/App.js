import "../styles/App.css";
import Search from "./Search.js";

function App() {
  return (
    <div className="App" data-testid="app">
      <img className="logo" src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg" />
      <Search />
    </div>
  );
}

export default App;
