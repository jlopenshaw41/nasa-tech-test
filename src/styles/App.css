.App {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.logo {
    margin: auto;
    width: 40%;
}