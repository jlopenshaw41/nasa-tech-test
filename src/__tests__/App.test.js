import { getByTestId, render } from '@testing-library/react';
import App from '../components/App';

describe("<App />", () => {
  it("renders as expected", () => {
    const { getByTestId } = render(<App />);
    const app = getByTestId("app");
    expect(app).toBeTruthy();

  })
})
