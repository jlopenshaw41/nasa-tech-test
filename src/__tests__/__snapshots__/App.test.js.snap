// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
    data-testid="app"
  >
    <img
      alt="NASA logo"
      class="logo"
      data-testid="logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <form
      class="search-form"
    >
      <input
        aria-label="Search"
        class="search-input"
        data-testid="search-input"
        type="text"
      />
      <button
        class="search-btn"
        data-testid="search-btn"
        type="submit"
      >
        Search
      </button>
    </form>
  </div>
</DocumentFragment>
`;
